/**
 * Copyright (c) 2015 - The CM Authors <legal@clickmatcher.com>
 *   All Rights Reserved.
 *
 * This file is CONFIDENTIAL -- Distribution or duplication of this material or
 * the information contained herein is strictly forbidden unless prior written
 * permission is obtained.
 */
package zbase;

message WebUIModuleConfig {

  optional bool is_public = 4;

  repeated string html_file = 1;

  repeated string css_file = 2;

  repeated string js_file = 3;

};

message WebUIRoute {

  optional bool is_public = 5;

  optional string path_prefix = 1;
  optional string path_match = 2;

  required string view = 3;

  repeated string require_module = 4;

};

message WebUIAppConfig {

  repeated WebUIRoute route = 1;

};
